recipes/barrnap/0.2/meta.yaml

```diff
@@ -0,0 +1,22 @@
+about:
+  home: https://github.com/tseemann/barrnap
+  license: GPL3
+  summary: 'Barrnap predicts the location of 5S, 16S and 23S ribosomal RNA genes in
+    Bacterial genome sequ It takes FASTA DNA sequence as input, and write GFF3 as
+    output.'
+build:
+  number: 0
+package:
+  name: barrnap
+  version: '0.2'
+requirements:
+  build: []
```

Comment:
Ok, this is more or less automatically generated I will tweak the script to omit it.
@johanneskoester is this a general style guideline for bioconda? I have seen several other places where they put these empty definitions in.